cypress/e2e/v1.0/pages/alcohol-use/alcohol-units.ts

```diff
@@ -0,0 +1,29 @@
+// export default (stepUrl: string, summaryPage: string, positionNumber: number) => {
```

Comment:
Are both the changes here and to `cypress/e2e/v1.0/pages/alcohol-use/alcohol-use.ts` intentional?